src/test/java/org/prebid/server/bidder/alkimi/AlkimiBidderTest.java

```diff
@@ -0,0 +1,285 @@
+        final HttpCall<BidRequest> httpCall = givenHttpCall(
+                null,
+                mapper.writeValueAsString(BidResponse.builder().build())
+        );
+        final Result<List<BidderBid>> result = alkimiBidder.makeBids(httpCall, null);
+
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).isEmpty();
+    }
+
+    @Test
+    public void makeBidsShouldReturnBidsForBannerAndVideoImps() throws JsonProcessingException {
+        final HttpCall<BidRequest> httpCall = givenHttpCall(
+                givenBidRequest(),
+                mapper.writeValueAsString(givenBidResponse())
+        );
+        final Result<List<BidderBid>> result = alkimiBidder.makeBids(httpCall, null);
+
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).contains(BidderBid.of(givenBannerBid(identity()), banner, null));
+        assertThat(result.getValue()).contains(BidderBid.of(givenVideoBid(identity()), video, null));
+    }
+
+    private static BidRequest givenBidRequest() {
+        return givenBidRequest(identity());
+    }
+
+    private static BidRequest givenBidRequest(Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
+        return givenBidRequest(identity(), impCustomizer);
+    }
+
+    private static BidRequest givenBidRequest(
+            Function<BidRequest.BidRequestBuilder, BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer
+    ) {
+        return bidRequestCustomizer.apply(BidRequest.builder()
+                .imp(List.of(givenBannerImp(impCustomizer), givenVideoImp(impCustomizer)))
+        ).build();
+    }
+
+    private static Imp givenBannerImp(Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
+        return impCustomizer.apply(Imp.builder()
+                .id(DIV_BANNER_ID)
+                .banner(Banner.builder()
+                        .format(Collections.singletonList(Format.builder()
+                                .w(300)
+                                .h(250)
+                                .build())
+                        ).build())
+                .ext(mapper.valueToTree(ExtPrebid.of(
+                        null,
+                        ExtImpAlkimi.builder()
+                                .token(PUB_TOKEN)
+                                .bidFloor(BigDecimal.valueOf(0.2))
+                                .pos(5)
+                                .build())))
+        ).build();
+    }
+
+    private static Imp givenVideoImp(Function<Imp.ImpBuilder, Imp.ImpBuilder> impCustomizer) {
+        return impCustomizer.apply(Imp.builder()
+                .id(DIV_VIDEO_ID)
+                .video(Video.builder()
+                        .w(1024)
+                        .h(768)
+                        .mimes(List.of("video/mp4"))
+                        .protocols(List.of(1, 2, 3, 4, 5))
+                        .build()
+                )
```

Comment:
We don't place ")" on new lines, place it on previous line. Please, check for other occurences.

Reply:
Fixed